config/locales/administrate.es.yml

```diff
@@ -6,6 +6,8 @@ es:
       destroy: Destruir
       edit: Editar
       show: Mostrar
+      new: Novaj
+      back: Reen
```

Comment:
@pablobm or @ismasan Could I get one of you two to look at this Spanish locale? (Even better the whole file!)